src/modules/ethereum/gas.js

```diff
@@ -10,7 +10,7 @@ export const cost = {
     // w narrative: 46733 - 56693
 
     PLACE_ORDER_GAS: 200000,
-    MATCH_ORDERS_GAS: 100000,
+    MATCH_ORDERS_GAS: 150000,
```

Comment:
why did you increase the gas price?

Reply:
match ran out of gas on rinkeby a couple of times. Unsure why b/c after increasing this limit the tx  excuted under 100k. This is gas limit only btw, the default given to end users - the actual gas consumption will depend on the transaction. (and the gas price is a different param, its default comes from the actual block gas price)